src/PackageVersions/Installer.php

```diff
@@ -48,6 +48,15 @@ public static function getVersion(string $packageName) : string
 
         return self::VERSIONS[$packageName];
     }
+
+    /**
+     * @throws \OutOfBoundsException if a version cannot be located
+     */
+    public static function getComposerVersion(string $packageName) : string
+    {
+        list($version) = explode('@', self::getVersion($packageName));
+        return $version;
```

Comment:
Add ltrim($version, 'v') here as suggested in https://github.com/Ocramius/PackageVersions/issues/21?